obs/obs-rest-api/src/main/java/com/adorsys/webank/obs/resource/OtpRequest.java

```diff
@@ -0,0 +1,16 @@
+package com.adorsys.webank.obs.resource;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OtpRequest {
+    @JsonProperty("otp")
+    private String otp;
+
+    public String getOtp() {
+        return otp;
+    }
+
+    public void setOtp(String otp) {
+        this.otp = otp;
+    }
+}
```

Comment:
I find it best that this file should go to the OBSServiceAPI, because here in the OBSRestApi, it will cause some circular dependency between RestApi, RestServer and ServiceApi